.github/workflows/osx.yml

```diff
@@ -0,0 +1,42 @@
+name: github-OSX
+
+on: [push, pull_request]
```

Comment:
On all branches? Or only for `master` and `develop`?

Reply:
We likely also want to test the release branches. I don't see a reason to exclude a branch only release `master` and `develop` should exist. (Of course, there is currently still a stale branch that should not be there.)